vcd/resource_vcd_dnat.go

```diff
@@ -2,6 +2,7 @@ package vcd
 
 import (
 	"fmt"
+	"github.com/hashicorp/terraform/helper/validation"
```

Comment:
github imports should be grouped together after stdlib ones

Reply:
fixed